component/sections/pricing/variant_b.js

```diff
@@ -7,65 +7,72 @@ function VariantB({
   title,
   description,
   plans,
-  sanityToken,
   hashKey,
   apiVersion,
   stripeSecretKey,
   stripePKey,
   NEXT_PUBLIC_DXP_STUDIO_ADDRESS,
 }) {
-  const [subscriptionProducts, setSubscriptionProducts] = React.useState([]);
   const [usePlan, setUsePlan] = React.useState(plans);
+  const [pKeyError, setPKError] = React.useState(false);
+  const comma = Intl.NumberFormat("en-us");
 
-  async function getPriceId(plans) {
-    let plansResponse = [];
-    let i = 0;
+  React.useEffect(() => {
+    async function getPriceId(plans) {
+      let i = 0;
+      for (; i < plans?.length; ) {
+        const productPayload = {
+          credentials: {
+            hashKey,
+            stripeSecretKey,
+            apiVersion,
+          },
+          StripeParams: {
+            id: `dxpstudio-pricing-${plans[i]?._key}-${plans[
+              i
+            ]?.planType?.replace(/ /g, "-")}-oneTimePrice-${plans[i]?.price}`,
+          },
+        };
```

Comment:
This payload needs to be updated as per https://github.com/webriq-pagebuilder/app/pull/63#discussion_r719403192